src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
"this might be a problem if abused" is a long shot from "this is being abused" when weighed against flexibility and utility. Abuse can be simply spotted and called out in review instead of clamping down a restrictive ruleset in a component.